klippy/extras/screws_tilt_adjust.py

```diff
@@ -92,8 +92,7 @@ def probe_finalize(self, offsets, positions):
                 self.gcode.respond_info(
                     "%s : x=%.1f, y=%.1f, z=%.5f" %
                     (name + ' (base)', coord[0], coord[1], z))
-                self.results.append({'name': name + ' (base)', 'x': coord[0],
-                    'y': coord[1], 'z': z, 'sign': 'CW', 'adjust':'00:00'})
+                self.results["screw%d" % (i + 1,)] = {'z': z, 'is_base': True}
```

Comment:
Unless I'm missing something, this drops the 'sign' and 'adjust' items from the dict.  Having dicts with entries that may or may not be present is a pain to code to.

-Kevin

Reply:
Indeed, I removed those from the item with `is_base=true` as they are pointless, but happy to add those back with some default value on them!

Reply:
@KevinOConnor I've now added those back (so all dict entries have the same keys) and updated the image on the initial post of this PR to reflect the change.